### ISSUE TITLE
[Bug] page 5557 "Permission Conflicts" - User Definied PermissionSets are missing

### ISSUE THREAD

**Issue by pri-kise** [microsoft/ALAppExtensions#23390]
Currently I'm unable to selected user defined permissionSets on the field `field("Permission Set ID"; PermissionSetId)` in `page 5557 "Permission Conflicts"`.

```al
field("Permission Set ID"; PermissionSetId)
{
    ApplicationArea = All;
    TableRelation = "Permission Set";
    Caption = 'Permission Set ID';
    ToolTip = 'Specifies the identifier for the permission set.';

    trigger OnValidate()
    begin
        EffectivePermissionsMgt.PopulatePermissionConflictsTable(EntitlementId, PermissionSetId, Rec);
        CurrPage.Update(false);
    end;
}
```

**Comment by pri-kise** [comment]
@JesperSchulz could you maybe take a look at this issue. It's probably caused by the TableRelation

**Comment by JesperSchulz** [comment]
I've created a bug for my team. We'll look into it.

**Comment by JesperSchulz** [comment]
Fixed in 23.0.

**Comment by JesperSchulz** [comment]
Thanks for reporting this. We agree, and we’ll publish a fix asap, either in an update for the current version or in the next major release. Please do not reply to this, as we do not monitor closed issues. If you have follow-up questions or requests, please create a new issue where you reference this one.

Build ID: 11807.
